src/Config/Config.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+/**
+ * Slim Framework (https://slimframework.com)
+ *
+ * @license https://github.com/slimphp/Slim-Console/blob/0.x/LICENSE.md (MIT License)
+ */
+
+declare(strict_types=1);
+
+namespace Slim\Console\Config;
+
+use ArrayAccess;
+
+use function array_key_exists;
+use function array_merge;
+use function ctype_lower;
+use function is_null;
+use function preg_replace;
+use function strtoupper;
+
+class Config implements ArrayAccess
+{
+    private const ENV_PREFIX = 'SLIM_CONSOLE_';
+
+    /**
+     * @var array<mixed>
+     */
+    private $default = [
+        'bootstrapDir'  => DIRECTORY_SEPARATOR . 'app',
+        'commandsDir'   => DIRECTORY_SEPARATOR . 'Application/Commands',
```

Comment:
Looking at how a DDD project is normally structured I would say that `Application/Commands` is better since you’ll typically only have only 3 root level folders being:
- Application
- Domain
- Infrastructure

Could even go one step further and do `Application/Console/Commands` since in there you would technically want to separate your Console and Web layers.